client/src/components/UserContext.tsx

```diff
@@ -68,6 +69,7 @@ export interface ISupportUser {
 export interface IAuthData {
   user: IUser | null
   supportUser: ISupportUser | null
+  setAuthData: (arg0: IAuthData) => void
```

Comment:
Why is this added to the context? My understanding is that the context is supposed to load the auth data and then pass it down to children components. I don't think children components should also be loading the auth data and then setting it on the context - that violates the one-direction data flow principle of React.

Reply:
@jonahkagan The issue I was running into was a bug on the DataEntry side of things for Audit Boards. The member names stopped showing in the header whenever the audit board forms were being submitted because it wasn't getting updated when the api was being called at that point. This just allows the DataEntry component to update the state from the top when it submits the member and the signoff forms.

Reply:
Gotcha. In that case, here are a few other suggestions:
- Hoist the auth state up a level and pass it into both the Header and the DataEntry components, adding a method to the auth context for the API request to set the audit board member names (I don't love this solution, since the auth data context is used for other user types as well)
- Make a separate header component for audit boards only and render within the DataEntry component. Manage the audit board state at the DataEntry component level. (This solution is more in line with the current approach. I'm pretty sure the reason we originally had to load the audit board within the DataEntry component instead of just using the auth context was so that we could reload it after setting the member names. Since the header is diverging in design for the audit boards anyway, this seems like it might make for simpler components)